internal/openshift/dashboards/metrics-dashboard.json

```diff
@@ -0,0 +1,1175 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "$$hashKey": "object:517",
+        "builtIn": 1,
+        "datasource": "-- Grafana --",
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "type": "dashboard"
+      }
+    ]
+  },
+  "description": "Provides information about the status of the OpenTelemetry Collector",
+    "editable": true,
+      "gnetId": 12553,
+        "graphTooltip": 0,
+          "id": 1,
```

Comment:
will this conflict with other dashboards?

Reply:
I removed the field. It seems it is not needed.